### PR TITLE
New package: mod_md-2.4.21

### PR DESCRIPTION
**Pull request by akierig**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

File: srcpkgs/mod_md/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'mod_md'
+pkgname=mod_md
+version=2.4.21
+revision=1
+build_style=gnu-configure
+configure_args="--with-apxs=/usr/bin/apxs --enable-werror"
```
Comment (paper42) [inline srcpkgs/mod_md/template:6]: we shouldn't be enabling Werror, compiler updates often add new warnings which would break the build for no real reason

**Comment by akierig** [comment]
I'm gonna mark this as x86/x86_musl/i686 only until I can update/test `apr` etc.

File: srcpkgs/mod_md/template (hunk 1 above)
Comment (akierig) [reply srcpkgs/mod_md/template]: fixed

**Comment by akierig** [comment]
Apache #43108 and APR #42972

**Comment by Duncaen** [comment]
Don't use install and remove messages as documentation, if void specific documentation is required add `README.voidlinux` file like other packages do.

**Comment by akierig** [comment]
> Don't use install and remove messages as documentation, if void specific documentation is required add README.voidlinux file like other packages do.

fixed, thanks!

**Comment by akierig** [comment]
I've decided against using apache `httpd(8)` on void--hence I won't need this package and am closing the PR.

**Pull request closed without merging**